docs/1.5/extensions/attributes.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+title: Attributes Extension
+description: The AttributesExtension allows HTML attributes to be added from within the document.
+---
```

Comment:
Missing `redirect_from: /extensions/attributes/`

Reply:
Good catch! This has been fixed.